firmware/screen/tools/scons_env_dump.py

```
@@ -0,0 +1,34 @@
+Import("env")
+import os
+
+project_dir = env['PROJECT_DIR']
+out = os.path.join(project_dir, '_log_scons_env.txt')
+
+with open(out, 'w', encoding='utf-8') as f:
+    f.write("=== scons_env_dump ===\n")
+    f.write(f"PROJECT_DIR: {project_dir}\n")
+    f.write("\n--- os.environ PATH ---\n")
+    f.write(os.environ.get('PATH','') + "\n")
+    scons_env = env['ENV']
```

Ruff (0.15.10)

[error] Undefined name `env`

(F821)